File: WebApp/src/app/shared/models/catalogue/catAddressPartner.model.ts
export class AddressPartner {
    id: string = "00000000-0000-0000-0000-000000000000";
    shortNameAddress: string = null;
    accountNo: string = null;
    shortName: string = null;
    taxCode: string = null;
    addressType: string = null;
    userCreated: string = null;
    datetimeCreated: string = null;
    userModified: string = null;
    datetimeModified: string = null;
    active: boolean = true;
    inactiveOn: string = null;
    userCreatedName: string = null;
    userModifiedName: string = null;
    contactPerson: string = null;
    tel: string = null;
    countryName: string = null;
    cityName: string = null;
    districtName: string = null;
    wardName: string = null;

    location: string = null;
    streetAddress: string = null;
    countryId: string = null;
    cityId: string = null;
    districtId: string = null;
    wardId: string = null;
    partnerId: string = null;
    index: number = null;
    
    constructor(data?: any) {
        const self = this;
        for (const key in data) {
            if (self.hasOwnProperty(key)) {
                self[key] = data[key];
            }
        }
    }
}

File: WebAPI/eFMS.API.SystemWeb/eFMS.API.Catalogue/eFMS.API.Catalogue.DL/Models/CatAddressPartnerModel.cs
using eFMS.API.Catalogue.Service.Models;

namespace eFMS.API.Catalogue.DL.Models
{
    public class CatAddressPartnerModel : CatAddressPartner
    {
        public string AccountNo { get; set; }
        public string ShortName { get; set; }
        public string TaxCode { get; set; }
        public string CountryName { get; set; }
        public string CityName { get; set; }
        public string DistrictName { get; set; }
        public string WardName { get; set; }
        public string UserCreatedName { get; set; }
        public string UserModifiedName { get; set; }
    }
}
